signingscript/test/test_sign.py

```diff
@@ -84,8 +84,6 @@ def context_die(*args, **kwargs):
         target_path = os.path.join(tmpdir, os.path.relpath(path, BASE_DIR))
         assert os.path.exists(target_path)
         assert os.path.isfile(target_path)
-        assert os.stat(target_path).st_uid == 0
-        assert os.stat(target_path).st_gid == 0
```

Comment:
I still think that we should test that the permissions inside the archive are correct.